fair/2.0.0/tests/numpy_tests/emissions_driven_test.py

```diff
@@ -261,3 +262,158 @@ def test_zero_emissions():
+                                    -0.304324144,\
+                                    -0.501633962,\
+                                    -0.904262779,\
+                                    -0.12278275])
+
+  ext_forcing_df = pd.DataFrame( data = ext_forcing_value_np,
+                                index = year_index_np,
+                                columns = ["External Forcing"])
+  
+  cfg = {'gas_params' : gas_params_df,'thermal_params' : thermal_params_df, 'ext_forcing' : ext_forcing_df}
+
+  out_dict = emissions_driven.run(inp_df, cfg)
+
+  emissions_df = out_dict['emissions']
+  C_df = out_dict['C']
+  RF_df = out_dict['RF']
+  T_df = out_dict['T']
+  alpha_df = out_dict['alpha']
+
+  C_compare_np = np.array([ [278.575556,\
+                            284.652403,\
+                            327.534824,\
+                            543.411648,\
+                            672.682666],\
+                            [769.601493,\
+                            959.837150,\
+                            2306.701551,\
+                            3499.109708,\
+                            3499.109708]])
+  T_compare_np = np.array([ -0.020142,\
+                            -0.025803,\
+                            0.172729,\
+                            0.880953,\
+                            1.835630])
+    #This should just be RF from the gasses themselves, not including external forcing
+  RF_compare_np = np.array([[0.011400,\
+                            0.130358,\
+                            0.904602,\
+                            3.717350,\
+                            4.919748],\
+                           [0.026254,\
+                            0.155020,\
+                            0.798028,\
+                            1.192708,\
+                            1.192708],\
+                           [-0.256119925,\
+                            -0.304324144,\
+                            -0.501633962,\
+                            -0.904262779,\
+                            -0.12278275],\
+                            [-0.21846593,\
+                            -0.01894614,\
+                            1.20099604,\
+                            4.00579522,\
+                            5.98967325]])
+  alpha_compare_np = np.array([ [0.125078,\
+                                0.123296,\
+                                0.123140,\
+                                0.119969,\
+                                0.065278],\
+                                [9.922729e-01,\
+                                8.111710e+01,\
+                                7.697883e+13,\
+                                2.245283e+106,\
+                                2.245283e+106]])
+
+  assert np.allclose(C_df.values, C_compare_np)
+  assert np.allclose(T_df.values, T_compare_np)
+  assert np.allclose(RF_df.values, RF_compare_np)
+  assert np.allclose(alpha_df.values, alpha_compare_np)
+
+  assert C_df.index.values == year_index_np
```

Comment:
```suggestion
  np.testing.assert_array_equal(C_df.index.values, year_index_np)
```

Will give better error messages on failure

Reply:
Done, will push.